scripts/json_device_add.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import hashlib
+
+
+def e(variable):
+    return os.environ.get(variable)
+
+
+json_path = os.path.join(e("BIN_DIR"), e("DEVICE_ID") + ".json")
+
+with open(os.path.join(e("BIN_DIR"), e("IMAGE_NAME")), "rb") as image_file:
+    image_hash = hashlib.sha256(image_file.read()).hexdigest()
+
+if not os.path.exists(json_path):
```

Comment:
No overwrite?
If I understand this correctly, the files should have been removed anyway?

Reply:
Ah, now I get it: You run this once per image ...

Reply:
Mark this as resolved...